<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumberForm</class>
 <widget class="QWidget" name="NumberForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>510</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>610</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_8">
     <property name="minimumSize">
      <size>
       <width>380</width>
       <height>480</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>480</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>361</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>表盘和LCD</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QDial" name="dial">
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber"/>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>LCD显示进制</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="radioBtn_Dec">
            <property name="text">
             <string>十进制</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioBtn_Oct">
            <property name="text">
             <string>八进制</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioBtn_Hex">
            <property name="text">
             <string>十六进制</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>361</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>水平</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSlider" name="horizonSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollBar" name="horizonScrollBar">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="horizonProgressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>361</width>
        <height>171</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string>Slider和QColor</string>
      </property>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>22</y>
         <width>91</width>
         <height>139</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>22</y>
         <width>51</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Red</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Green</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Blue</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Alpha</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>66</x>
         <y>22</y>
         <width>191</width>
         <height>141</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QSlider" name="alphaSlider">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>102</y>
          <width>161</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="greenSlider">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>38</y>
          <width>161</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="redSlider">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>6</y>
          <width>161</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="blueSlider">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>161</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_9">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>480</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>480</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>垂直</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSlider" name="verticalSlider">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollBar" name="verticalScrollBar">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="verticalProgressBar">
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="value">
            <number>24</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber_hour">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber_min">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber_second">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="elapstimelabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>32</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exitBtn">
        <property name="text">
         <string>exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
